src/main/java/com/unb/tracker/web/CourseController.java

```diff
@@ -125,31 +125,43 @@ public String courseSave(@ModelAttribute Course course, Model model, Principal p
         if (user == null || !user.getHasExtendedPrivileges()) {
             throw new BadRequestException();
         }
+        course.setInstructor(user);
 
         courseValidator.validate(course, bindingResult);
         if (bindingResult.hasErrors()) {
             redir.addFlashAttribute("courseCreationError", bindingResult.getFieldError().getCode());
             redir.addFlashAttribute("courseName", course.getName());
             redir.addFlashAttribute("sectionName", course.getSection());
-            redir.addFlashAttribute("startDate", course.getStartDate().toString());
+            if(course.getStartDate().toString().equals("1970-01-01")) {
```

Comment:
Is there something somewhere that converts empty value to the beginning of Unix time or something?

EDIT: Nvm saw further down